sdk/keyvault/azure-security-keyvault-keys/src/samples/java/com/azure/security/keyvault/keys/KeyAsyncClientJavaDocCodeSnippets.java

```diff
@@ -497,6 +498,28 @@ public void listKeyVersions() {
         // END: com.azure.security.keyvault.keys.async.keyclient.listKeyVersions
     }
 
+    /**
+     * Generates code samples for using {@link KeyAsyncClient#getRandomBytes(int)} and
+     * {@link KeyAsyncClient#getRandomBytesWithResponse(int)}.
+     */
+    public void getRandomBytes() {
+        KeyAsyncClient keyAsyncClient = createAsyncClient();
+
+        // BEGIN: com.azure.security.keyvault.keys.KeyAsyncClient.getRandomBytes#int
+        int amount = 16;
+        keyAsyncClient.getRandomBytes(amount)
+            .subscribe(randomBytes ->
+                System.out.printf("Retrieved %d random bytes: %s%n", amount, Arrays.toString(randomBytes.getBytes())));
+        // END: com.azure.security.keyvault.keys.KeyAsyncClient.getRandomBytes#int
+
+        // BEGIN: com.azure.security.keyvault.keys.KeyAsyncClient.getRandomBytesWithResponse#int
+        int amountOfBytes = 16;
```

Comment:
Same here, should this variable be named `count` for consistency? I'm not sure if this is generated code or not - but figured I'll ask here and it would apply everywhere else

Reply:
You are right, we should name this `count`.